jar/src/cljc/sixsq/slipstream/client/api/utils/http_utils.cljc

```diff
@@ -0,0 +1,16 @@
+(ns sixsq.slipstream.client.api.utils.http-utils)
+
+(def http-lib-insecure-key :kvlt.platform/insecure?)
+
+(defn- process-insecure
+  [req]
+  (if (contains? req :insecure?)
+    (-> req
+      (merge {http-lib-insecure-key (:insecure? req)})
+      (dissoc :insecure?))
+    req))
+
+(defn process-req
```

Comment:
This function doesn't really seem to be useful.  It just calls the `process-insecure` with the same argument.  Probably it would be better to just rename `process-insecure` to `process-req`.